app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
```

Comment:
The import statements should be grouped according to the checklist: standard library imports first, followed by local application/library-specific imports. In this case, `import datetime` should come before the import from `app.errors` .